src/openforms/registrations/contrib/objects_api/tests/test_backend.py

```diff
@@ -220,3 +226,199 @@ def test_registration_works_after_failure(self, m: requests_mock.Mocker):
 
         self.assertEqual(registration_data.pdf_url, "https://example.com")
         self.assertEqual(registration_data.csv_url, "")
+
+
+class ObjectsAPIBackendVCRTests(OFVCRMixin, TestCase):
+    VCR_TEST_FILES = Path(__file__).parent / "files"
+
+    def setUp(self):
+        super().setUp()
+
+        config_patcher = patch(
+            "openforms.registrations.contrib.objects_api.models.ObjectsAPIConfig.get_solo",
+            return_value=ObjectsAPIConfig(),
+        )
+        self.mock_get_config = config_patcher.start()
+        self.addCleanup(config_patcher.stop)
+
+        self.objects_api_group = ObjectsAPIGroupConfig.objects.create(
```

Comment:
You may need to rebase, but you can now use `ObjectsAPIGroupConfigFactory.create(for_test_docker_compose=True)`